lib/src/utils/widgets/linked_in_buttons.dart

```diff
@@ -11,45 +11,45 @@ class LinkedInButtonStandardWidget extends StatelessWidget {
     required this.onTap,
     this.iconHeight = 30,
     this.iconWeight = 30,
-    this.iconAssetPath = 'assets/linked_in_logo.png',
-    this.buttonText = 'Sign in with LinkedIn',
-    this.buttonColor = Colors.white,
-    this.textPadding = const EdgeInsets.all(4),
+    this.buttonText = '',
+    this.textStyle,
+    this.textPadding = const EdgeInsets.symmetric(horizontal: 4),
```

Comment:
again possible breaking change for some of the users, because of this reason clients should always pass padding that are suitable for them. Please revert a change